docs/services/slam/cartographer/_index.md

```diff
@@ -0,0 +1,366 @@
+| `map_rate_sec` | int | Optional | Rate of <file>/map</file> generation *(seconds)*. <ul> Default: `60`. </ul> |
+| `data_rate_msec` | int | Optional | Rate of <file>/data</file> collection from `sensors` *(milliseconds)*. <ul>Default: `200`.</ul> |
+| `port` | string | Optional | Port for SLAM gRPC server. If running locally, this should be in the form "localhost:<PORT>". If no value is specified a random available port is assigned. |
+| `delete_processed_data` | bool | Optional | <p>Setting this to `true` helps to reduce the amount of memory required to run SLAM.</p> <ul> `true`: sensor data is deleted after the SLAM algorithm has processed it. </ul><ul> `false`: sensor data is not deleted after the SLAM algorithm has processed it. </ul> |
+| `config_params` |  map[string] string | Optional | Parameters specific to the `model` of SLAM library. |
+
+{{% alert title="Caution" color="caution" %}}
+
+- If `use_live_data: true`, `delete_processed_data: true` by default.
+- If `use_live_data: false`, `delete_processed_data: false` by default.
+
+Setting `delete_processed_data: true` and `use_live_data: false` is invalid and will result in an error.
+{{% /alert %}}
+
+{{% /tab %}}
+{{% tab name="config_params" %}}
+
+Adjust these parameters to fine-tune the algorithm `cartographer` utilizes in aspects like submap size, mapping update rate, and feature matching details:
+
+| Parameter Mode | Description | Inclusion | Default Value | Notes |
+| -------------- | ----------- | --------- | ------------- | ----- |
+| `mode` | `2d` | **Required** | None | |
+| `optimize_every_n_nodes` | How many trajectory nodes are inserted before the global optimization is run. | Optional | `3` | |
+| `num_range_data` | Number of measurements in each submap. | Optional | `100` | |
+| `missing_data_ray_length` | Replaces the length of ranges that are further than max_range with this value. | Optional | `25` | Nominally set to max length. |
+| `max_range` | Maximum range of valid measurements. | Optional | `25` | |
+| `min_range` | Minimum range of valid measurements. | Optional | `0.2` | |
+| `max_submaps_to_keep` | Number of submaps to use and track for localization. | Optional | `3` | Only for [LOCALIZING mode](#mapping-modes). |
+| `fresh_submaps_count` | Length of submap history considered when running SLAM in updating mode. | Optional | `3` | Only for [UPDATING mode](#mapping-modes). |
+| `min_covered_area` | The minimum overlapping area, in square meters, for an old submap to be considered for deletion. | Optional | `1.0` | Only for [UPDATING mode](#mapping-modes). |
+| `min_added_submaps_count` | The minimum number of added submaps before deletion of the old submap is considered. | Optional | `1` | Only for [UPDATING mode](#mapping-modes). |
```

Comment:
[nit] The links to LOCALIZING mode and UPDATING mode don't link to the mode descriptions. They link to the top of the page

Reply:
Have fixed!